app/tests/conftest.py

```diff
@@ -480,3 +480,14 @@ def authenticated_staff_user(client):
         user=user,
     )
     return user
+
+
+@pytest.fixture
+def user_with_totp():
+    def _make_user_with_totp(is_staff=False):
```

Comment:
Is there a need to define this as a fixture? It is only used in one test. You could have defined `_make_user_with_totp` just in the one test that it is used in.

Reply:
I thought we might want to use this for other tests in the future, but at the moment, it can also just be defined in the one test, yes.